src/screens/NewsScreen/Search.js

```diff
@@ -29,7 +29,7 @@ const Search = ({animatedValue}) => {
         onPress={handleOnContainerPress}>
         <View style={styles.wrapperSearch}>
           <View style={styles.wrapperIcon}>
-            <MemoIc_search width={16.67} height={16.67} />
+            <MemoIc_search width={16.67} height={16.67} fill={COLORS.gray300} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is to add a fill color to the `MemoIc_search` SVG icon. This seems fine as long as `COLORS.gray300` is defined and is the desired color for this icon. However, it's important to ensure that this color change doesn't affect the visibility or aesthetics of the icon in its various usage contexts.

```diff
- <MemoIc_search width={16.67} height={16.67} />
+ <MemoIc_search width={16.67} height={16.67} fill={COLORS.gray300} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->